torchchat/utils/quantize.py

```diff
@@ -890,7 +901,7 @@ def quantized_model(self) -> nn.Module:
     sys.modules["torchao_experimental_quant_api"] = torchao_experimental_quant_api
     torchao_experimental_quant_api_spec.loader.exec_module(torchao_experimental_quant_api)
     from torchao_experimental_quant_api import Int8DynActIntxWeightQuantizer
-    ao_quantizer_class_dict["linear:a8wxdq"] = Int8DynActIntxWeightQuantizer
+    quantizer_class_dict["linear:a8wxdq"] = Int8DynActIntxWeightQuantizer
```

Comment:
Any reason we can't put this inside of the `quantizer_class_dict`?

Reply:
It seems cleaner to try import `Int8DynActIntxWeightQuantizer` first and fallback to `ErrorHandler`, then assign this handler to `linear:a8wxdq` in `quantizer_class_dict`.

Reply:
I think if this can go away soon I don't have a strong opinion and can live with this.